core/controllers/domain_objects_validator.py

```diff
@@ -29,11 +29,10 @@
 from core.domain import exp_domain
 import python_utils
 
-from typing import Any, Dict # isort:skip  pylint: disable=wrong-import-order, wrong-import-position, unused-import, import-only-modules
+from typing import Dict, Optional, Union # isort:skip
```

Comment:
This file was filled with usages of unexplained Any. So I decided to keep this file in denylist only, removed type annotations with unexplained Any, but kept the remaining annotations.